### DISCUSSION TITLE
custom plugin access to Kong built-in module

### DISCUSSION THREAD

**Discussion by ponsakorn30214** [Kong/kong#12564] [Help] [answered]
Hi all,
I've been developing a custom plugin which need to send info to an endpoints in a batch.
After looking at http-log plugin, I saw that there is a helper function Queue (kong.tools.queue) being use and it's seems to be easier to use it than implementing my own queue.
Can I import it as a module from to my custom plugin?

**Comment by chronolaw** [comment, marked as answer]
Of course, all modules in kong's code base can be used in custom plugin.
